string[]  joint_name                        # Unique identifier for the joint
float64[] position                          # rad or m
float64[] velocity                          # rad/s or m/s
float64[] acceleration                      # rad/s/s or m/s/s
float64[] effort                            # Nm or N
